api/src/pcapi/core/offers/repository.py

```diff
@@ -689,6 +689,29 @@ def get_offer_by_id(offer_id: int) -> models.Offer:
         raise exceptions.OfferNotFound()
 
 
+def get_filtered_stocks(
+    offer_id: int,
+    stocks_limit_per_page: int,
+    date: datetime.date | None = None,
+    time: datetime.time | None = None,
+    price_category_id: int | None = None,
+) -> list[models.Stock]:
```

Comment:
```suggestion
def get_filtered_stocks(
    offer_id: int,
    date: datetime.date | None = None,
    time: datetime.time | None = None,
    price_category_id: int | None = None,
    stocks_limit_per_page: int = LIMIT_STOCKS_PER_PAGE,
) -> list[models.Stock]:
```
te permet de supprimer la surcouche que t'as écrit dans `api.py`

Reply:
C'est OK de mettre LIMIT_STOCKS_PER_PAGE dans repository plutôt que API ?